beancount/ops/pad_test.py

```diff
@@ -45,6 +45,8 @@ def test_pad_simple(self, entries, errors, __):
           2013-05-03 balance Assets:Checking                                 172.45 USD
 
         """, entries)
+        self.assertEqual(entries[3].flag, 'P')
+        self.assertFalse(any([ p.meta is None for p in entries[3].postings]))
```

Comment:
The right thing to check is that all postings have a dictionary in the meta field. Also you don't need to create a list, a generator comprehensions works best here. 
```python
self.assertTrue(all(isinstance(p.meta, dict) for p in entries[3].postings))
```